crates/metadata/src/lib.rs

```diff
@@ -131,15 +115,15 @@ impl InkProject {
         registry: PortableRegistry,
     ) -> Self {
         Self {
-            version: Default::default(),
+            version: METADATA_VERSION,
             layout,
             spec,
             registry,
         }
     }
 
     /// Returns the metadata version used by the contract.
-    pub fn version(&self) -> &MetadataVersion {
+    pub fn version(&self) -> &u64 {
```

Comment:
Will this change result in the version being output as a number?

So `{version: 5}` instead of `{version: '5'}`?

Reply:
Yes, this changes the string into the integer. All good there

Reply:
$DOP (Data Ownership Protocol) dop.org - Currently in the last rounds of their private sale ending in a day and few hours, listing on binance this month of march, the private sale price is $0.08 but don't be surprised that it will push to $5 in few days after listing because is a high demand project.
Access the private sale here: privatesale.dop.org
  access code:3656924464